src/admin-events/admin-events.spec.js

```diff
@@ -2,13 +2,16 @@ const ethers = require('ethers');
 const { getAddress } = require('@uma/contracts-node');
 const { createTransactionEvent } = require('forta-agent');
 
-const { createAlert, handleTransaction } = require('./admin-events');
+const { createAlert, handleTransaction, initialize } = require('./admin-events');
 
 const CHAIN_ID = 1;
 const votingAddressPromise = getAddress('Voting', CHAIN_ID);
 
 // Tests
 describe('admin event monitoring', () => {
+  // Initialize the Handler
+  initialize();
```

Comment:
The `initialize` function is `async`, which means it will return a promise.  If we don't `await` that promise, we have no guarantee that it will execute to completion before other code is executed.

I assume that the anonymous arrow function passed as the second argument to the `describe` function on line 11 could be changed (see suggestion on that line).